### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by da-i**

### CODE REVIEW

File: main.nf [hunk 1]
```diff
@@ -236,6 +237,26 @@ workflow {
         reference_mmi = PrepareGenome.out.mmi_combi.collect()
         consensus = CygnusAlignedConsensus(read_fastq, reference_mmi)
     }
+    else if (params.consensus_method == "Cygnus2_aligned") {
+        log.info """Cygnus2_aligned consensus generation method selected."""
+        log.info """We will align against the provided primer."""
+        if (params.reference == "" || backbone_file == "") {
+            log.error \
+            """Please provide reference genome and backbone file for Cygnus)aligned method.
```
Comment (marcpaga) [inline main.nf:245]: There's a `)` in the text that should not be there.

File: main.nf [hunk 2]
```diff
@@ -236,6 +237,26 @@ workflow {
...
+            reference genome can be provided with --reference and was: '${params.reference}' 
+            backbone file can be provided with --backbone_file or --backbone and were: '${params.backbone_file}' or '${params.backbone}'
+            """
```
[the remark this answers is not in the document]
Comment (da-i) [reply main.nf:248]: Its true that one will always be empty, but i wanted to limit the amount of gate clauses in the code. @groodri  What would your preference be?

[the remark this answers is not in the document]
Comment (groodri) [reply main.nf:248]: I would be in favour of validating each argument independently

File: main.nf (hunk 1 above)
[the remark this answers is not in the document]
Comment (da-i) [reply main.nf:245]: The magic of multicursor backfired hard ;).

**Review by groodri — APPROVED** [review]

**Review by marcpaga — APPROVED** [review]

**Pull request merged**